--- lib/VMware/share/VMware/VICommon.pm	2015-02-09 19:46:18.000000000 -0200
+++ lib/VMware/share/VMware/VICommon.pm.new	2015-05-27 17:00:12.825355474 -0300
@@ -587,15 +587,15 @@
    my $url = shift;
    my %supportedapiversions;
    if ($url =~ s|http(s?)://(.*)/sdk.*|http$1://$2/sdk/vimService.wsdl|i) {      
-      if ($1 eq "s") {
-         eval {  
-            require Crypt::SSLeay;  
-            Crypt::SSLeay->import();
-         };
-         if ($@) {
-            die "Crypt::SSLeay is required for https connections, but could not be loaded: $@";
-         }
-      }
+#      if ($1 eq "s") {
+#         eval {  
+#            require Crypt::SSLeay;  
+#            Crypt::SSLeay->import();
+#         };
+#         if ($@) {
+#            die "Crypt::SSLeay is required for https connections, but could not be loaded: $@";
+#         }
+#      }
 
       my $temp_addr = $2;
       if ($temp_addr =~ /:/) {
@@ -664,23 +664,23 @@
 
 sub query_server_version {
    BEGIN {
-      #To remove SSL Warning, switching from IO::Socket::SSL to Net::SSL
-      $ENV{PERL_NET_HTTPS_SSL_SOCKET_CLASS} = "Net::SSL";
+#      #To remove SSL Warning, switching from IO::Socket::SSL to Net::SSL
+#      $ENV{PERL_NET_HTTPS_SSL_SOCKET_CLASS} = "Net::SSL";
 	  #To remove host verification
       $ENV{PERL_LWP_SSL_VERIFY_HOSTNAME} = 0;
    }
    my $url = shift;
    if ($url =~ s|http(s?)://(.*)/sdk.*|http$1://$2/sdk/vimService.wsdl|i) {
-      # bug 288336
-      if ($1 eq "s") {
-         eval {
-            require Crypt::SSLeay;
-            Crypt::SSLeay->import();
-         };
-         if ($@) {
-            die "Crypt::SSLeay is required for https connections, but could not be loaded: $@";
-         }
-      }
+#      # bug 288336
+#      if ($1 eq "s") {
+#         eval {
+#            require Crypt::SSLeay;
+#            Crypt::SSLeay->import();
+#         };
+#         if ($@) {
+#            die "Crypt::SSLeay is required for https connections, but could not be loaded: $@";
+#         }
+#      }
 
       my $temp_addr = $2;
       if ($temp_addr =~ /:/) {
